### PR TITLE
Get process definition name and key implemented

### PR DESCRIPTION
**Pull request by danishjamal104**
Implemented the api for fetching `process definition name and key`.

*Sample Response*
```JSON
{
   "bulk_processor-ibank-usa":[
      2251799813685998,
      2251799814125425
   ],
   "international_remittance_payee_process-ibank-india":[
      2251799813686276,
      2251799814069864
   ],
   "international_remittance_payer_process-ibank-usa":[
      2251799813686414,
      2251799814069794
   ],
   "international_remittance_payer_process-ibank-india":[
      2251799813686138,
      2251799814070206
   ],
   "international_remittance_payee_process-ibank-usa":[
      2251799813686068,
      2251799814070344
   ]
}
```

### CODE REVIEW

**Comment by danishjamal104** [comment]
I have resolved the conflict please review. Thanks :)

File: src/main/java/org/mifos/ops/zeebe/camel/routes/OperationsRouteBuilder.java [hunk 1]
```diff
@@ -121,6 +128,133 @@ public void configure() {
 
                 });
 
+        /**
+         * Get the process definition key and name
+         *
+         * equivalent es query: GET _search
```
Comment (SubhamPramanik) [inline src/main/java/org/mifos/ops/zeebe/camel/routes/OperationsRouteBuilder.java:134]: Remove this equivalent ES query from here to PR description. Over here just add the request and response.

**Review by SubhamPramanik — CHANGES_REQUESTED** [review]
LGTM. Just change the comment and do some error handling/null checks when calling the ES client and when parsing the response.

File: src/main/java/org/mifos/ops/zeebe/camel/routes/OperationsRouteBuilder.java (hunk 1 above)
Comment (danishjamal104) [reply src/main/java/org/mifos/ops/zeebe/camel/routes/OperationsRouteBuilder.java:134]: sure

**Pull request merged**